controllers/form-instances.ts

```diff
@@ -113,4 +120,51 @@ formInstanceRouter.delete(
   },
 );
 
+formInstanceRouter.post(
+  '/custom-form-type/create',
+  async (req: Request, res: Response) => {
+    if (!req.body.name) {
+      throw new HttpError('No name was provided.', 400);
+    }
+
+    const formType = await createCustomFormType(req.body.name);
+    res.status(201).send({
+      id: formType.id,
+    });
+  },
+);
+
+formInstanceRouter.get(
+  '/custom-form-type/list',
+  async (req: Request, res: Response) => {
+    const types = await fetchCustomFormTypeList();
+    res.status(200).send({ types });
+  },
+);
+
+formInstanceRouter.post(
+  '/custom-form-instance/create',
```

Comment:
this does not create form instances but rather custom forms, please get rid of the instance